src/utils/authentication/fakeDB.ts

```diff
@@ -0,0 +1,457 @@
+import { nanoid } from "nanoid";
```

Comment:
This file will be deleted once the real server is ready. Types and TODOs here just serve as "spec".

@matteyu See here for the latest changes.